### ISSUE TITLE
Carbon Calculator: Accessibility - Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by BrianGibbVS** [visitscotland/vs-component-library#246]
### Raised by
Brian Gibb

### Describe the bug
In the Carbon Calculator, the **next** and **results** button is greyed out until a mode of transport is selected by the user. This "disabled state" fails an Accessibility colour contrast check as the ratio of foreground to background falls below 4.5:1.

### To Reproduce

1. Go to Carbon Calculator page
2. Conduct an Accessibility Scan on each of the Carbon Calculator steps.
3. See screenshots.

### Expected behavior
As per [WCAG guildlines](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast), contrast between foreground and background colours should meet WCAG 2 AA minimum contrast ratio thresholds.

### Page URL
https://www.visitscotland.com/travel-planning/responsible-tourism/carbon-calculator

### Environment
All

### Browser
All

### Device and version
N/A

### Impact on end user
Visually impaired users of the site may not be able to read the text on the button and thus understand its purpose.

### Impact on product development
N/A

### Additional context
N/A
